recipe/bld.bat

```diff
@@ -11,13 +11,13 @@ cmake %CMAKE_ARGS% -GNinja ^
       -DCMAKE_INSTALL_LIBDIR=%LIBRARY_LIB% ^
       -DCMAKE_INSTALL_INCLUDEDIR=%LIBRARY_INC% ^
       -DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
+      -DCMAKE_MODULE_PATH=%LIBRARY_PREFIX%\mingw-w64\lib\pkgconfig ^
```

Comment:
```suggestion
      -DCMAKE_MODULE_PATH=%LIBRARY_PREFIX%/mingw-w64/lib/pkgconfig ^
```
Use forward slashes `/` in your paths, see https://stackoverflow.com/a/13742066.
I think it will unblock win64